Fix windows tip appearing in code block

Ste1io opened the pull request:
### Why:

Closes #12074 

### What's being changed:

Removed unnecessary indentation for the tip.

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

Ste1io committed:
Fix windows tip appearing in code block

Ste1io committed:
Merge branch 'main' into patch-4

inderashah007 commented on content/get-started/using-git/splitting-a-subfolder-out-into-a-new-repository.md:
````diff
@@ -27,24 +27,24 @@ If you create a new clone of the repository, you won't lose any of your Git hist
   ```
 5. To filter out the subfolder from the rest of the files in the repository, run [`git filter-repo`](https://github.com/newren/git-filter-repo), supplying this information:
-    - `FOLDER-NAME`: The folder within your project where you'd like to create a separate repository.
+   - `FOLDER-NAME`: The folder within your project where you'd like to create a separate repository.
````
```suggestion
   - `FOLDER-NAME`: The folder within your project where you'd like to create a separate repository.
```

StelioKontosXBL replied:
I used 3 spaces per [GitHub Flavored Markdown Spec 5.2 ex 274](https://github.github.com/gfm/#example-274).

StelioKontosXBL replied:
My apologies, just looked again and realized your revision is correct; had it in my head I had already set it to 3 spaces, but I guess I missed that line. Thanks.

Ste1io committed:
Corrected remaining indentation and spacing

After reviewing a few other doc pages, adjusted indentation accordingly and spaced the numbered list items for consistency.

ramyaparimi commented:
@StelioKontosXBL 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

mchammer01 reviewed (approved):
@StelioKontosXBL - thanks for fixing the issue with the Windows tip ✨

mchammer01 committed:
Merge branch 'main' into patch-4
